graph-manager-docs/source/managed-federation/overview.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Managed federation overview
+sidebar_title: Overview
+---
+
+Apollo Graph Manager provides free **managed federation** features for data graphs that use [Apollo Federation](https://www.apollographql.com/docs/apollo-server/federation/introduction/).
+
+With managed federation, your [gateway](https://www.apollographql.com/docs/apollo-server/federation/gateway/) is no longer responsible for combining your [implementing service](https://www.apollographql.com/docs/apollo-server/federation/implementing-services/) schemas into a single federated schema. Instead, your implementing services push their schemas to Graph Manager, which combines them. Your gateway then fetches the federated schema and associated configuration from Graph Manager:
+
+```mermaid
+graph BT;
+  subgraph " "
+  serviceA[Products service];
+  serviceB[Reviews service];
+  gateway([Gateway]);
+  end
+  graphManager{{Apollo Graph Manager}};
+  serviceA & serviceB -. Pushes schema .-> graphManager;
+  gateway -. Regularly polls for combined schema .-> graphManager;
+  class graphManager secondary;
+```
+
+This architecture helps your organization safely validate, coordinate, deploy, and monitor changes to your graph:
+
+* You can modify service schemas (and even add or remove entire services from your graph) _without_ needing to redeploy your gateway. Your gateway polls Graph Manager regularly for these changes and updates its query plans dynamically.
+
+* If you accidentally push an _invalid_ schema change, Graph Manager continues to provide the most recent valid schema to your gateway. This prevents unintended downtime.
+
+* You can disable schema introspection for all of your implementing services, because the gateway no longer requires introspection to fetch schemas. This improves security.
```

Comment:
I'm not really sure what this means. Your implementing service should always be inaccessible from the outside world (it's a huge security risk for it not to be), so do you mean you can remove the `_service { sdl }` resolver from your service? I don't think we would recommend that 🤔

Reply:
Yeah, I guess there really isn't much of a security improvement here, huh. I'll cut this one.